### ISSUE TITLE
Multiple <h1>s on static site /support

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#4515]
Navigate to `/support` on the static site. There are multiple headers on that page that are `<h1>`s. Those headers should be a `<h2>`

**Comment by georgehager** [comment]
@emyl3 @emmastephenson @lauraykerr Just want to confirm an a11y question on the support page. On the iframe video, I'm seeing a "This potential issue needs your review..."  and "Check that captions is available for the element". I believe the link to a transcript below satisfies the requirement for captions, but would love confirmation. Thanks!

![Screen Shot 2022-11-23 at 8 26 11 AM](https://user-images.githubusercontent.com/89797785/203558587-bbeb11e0-cc15-4861-8b8f-fade553753ca.png)

**Comment by georgehager** [comment]
Also confirming that this "potential issue" is not fixable, because the aria-label in question is autogenerated by the iframe element for the youtube video:

![Screen Shot 2022-11-23 at 8 33 01 AM](https://user-images.githubusercontent.com/89797785/203559683-63c03ada-c16c-4d4e-9b8a-1cefac3382ae.png)

**Comment by emyl3** [comment]
> Also confirming that this "potential issue" is not fixable, because the aria-label in question is autogenerated by the iframe element for the youtube video. I believe we've looked into this before and determined it is out of our control. cc: @emmastephenson
> 
> ![Screen Shot 2022-11-23 at 8 33 01 AM](https://user-images.githubusercontent.com/89797785/203559683-63c03ada-c16c-4d4e-9b8a-1cefac3382ae.png)

Yes, you are right, George! 😸

**Comment by emyl3** [comment]
> @emyl3 @emmastephenson @lauraykerr Just want to confirm an a11y question on the support page. On the iframe video, I'm seeing a "This potential issue needs your review..." and "Check that captions is available for the element". I believe the link to a transcript below satisfies the requirement for captions, but would love confirmation. Thanks!
> 
> ![Screen Shot 2022-11-23 at 8 26 11 AM](https://user-images.githubusercontent.com/89797785/203558587-bbeb11e0-cc15-4861-8b8f-fade553753ca.png)

I think the captions and transcripts are different but I believe we are still OK because we have the auto generated captions provided by Youtube!

**Comment by emmastephenson** [comment]
Yep! Elisa is right, we can't fix issues within the iframe element, and YouTube covers captions.
